recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: dask-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
-  md5: 581c3f23c46eea79a1b68aed4579d57a
+  md5: 6acbd2e22487620a8792bec32251ad63
 
 build:
   number: 1
```

Comment:
Just to be clear, that is what you should normally do, but since we have now built this version on conda-forge, it makes sense to live with this number now.